### DISCUSSION TITLE
pyfor module error

### DISCUSSION THREAD

**Discussion by safisoomro** [weecology/DeepForest#593] [General]
I get an error for pyfor module even I have installed it multiple times, what could be the issue@

**Comment by ethanwhite** [comment]
Can you please provide the code that resulted in the error and the full error message? Thanks

**Comment by safisoomro** [comment]
I found the train.py code in your previous repository.. I read your paper
as well but it is semi supervised... I was actually confused that you have
converted .las images in RGB then trained in to your model.. tell me if i
am wrong...

I have LiDAR data from my organization, now how can I use your model over
that data.. Can you briefly guide me? Thanks

2024년 1월 8일 (월) 오후 4:38, Ethan White ***@***.***>님이 작성:

> Can you please provide the code that resulted in the error and the full
> error message? Thanks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8051382>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALV2GYXGRMZFQ34X4OC6OKTYNQHF3AVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANJRGM4DE>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
*Associate Professor *
*Department of Computer Science*
*Quaid-e-Awam University of Engineering Science and Technology Nawabshah
Sindh Pakistan.*
***@***.*** ***@***.***>*

**Comment by ethanwhite** [comment]
Our model works on RGB data. We used LiDAR data to create unsupervised estimates of trees for pre-training. So, if you are looking to do crown delineation using LiDAR our package isn't what you're looking for. The lidR package is one decent option for what you are looking to do https://r-lidar.github.io/lidRbook/index.html

**Comment by safisoomro** [comment]
YesI get it now. But you did not add that step where you have done
pertaining on LiDAR data before Performing training on RGB.

**Comment by ethanwhite** [comment]
> But you did not add that step where you have done pertaining on LiDAR data before Performing training on RGB.

It's described in a number of the key papers. Here's the relevant section from the DeepForest software paper:

https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.13472#mee313472-sec-0003-title

Or did you mean that we don't provide code for doing the pretraining on the LiDAR data separately?

**Comment by safisoomro** [comment]
Yes, you are right. I mean you didn't provide code for that.

**Comment by ethanwhite** [comment]
Thanks for clarifying.

@bw4sz - do we have the code we used for creating/pretraining on the LiDAR-based crowns anywhere on GitHub?

**Comment by safisoomro** [comment]
Moreover, I am trying to run train.py file available in DeepLidar..
how can i resolve this error
File "C:\Users\shsoaa\Desktop\Forest\DeepLidar-master\train.py", line 8, in
<module>
    from comet_ml import Experiment, predictor
ImportError: cannot import name 'predictor' from 'comet_ml'
(C:\Users\shsoaa\Desktop\Forest\DeepLidar-master\demo\venv\lib\site-packages\comet_ml\__init__.py)

2024년 1월 8일 (월) 오후 9:27, Ethan White ***@***.***>님이 작성:

> Thanks for clarifying.
>
> @bw4sz <https://github.com/bw4sz> - do we have the code we used for
> creating/pretraining on the LiDAR-based crowns anywhere on GitHub?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8055405>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALV2GYRR3GHAS67MUQL6CX3YNRJB5AVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANJVGQYDK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

*Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
*Associate Professor *
*Department of Computer Science*
*Quaid-e-Awam University of Engineering Science and Technology Nawabshah
Sindh Pakistan.*
***@***.*** ***@***.***>*

**Comment by ethanwhite** [comment]
DeepLidar is a (very) out of date early implementation. It may have the code we used to do some of that early pretraining work (we'll have to wait to hear from @bw4sz), but I definitely wouldn't expect it to run any more. I'll go mark that repo has archived to avoid folks trying to run it.

**Reply by ethanwhite** [reply]
Marked as archived now. Sorry for the confusion @safisoomro

**Comment by safisoomro** [comment]
No problem, thank you!  Ill follow Deepforest repo but it would be very
helpful if I get thw pertaining code for LiDAR

**Comment by safisoomro** [comment]
Hello,

I am wondering if you can show us validation on LiDAR point cloud instead
of RGB dataset.. because this model is pre-trained on LiDAR data. So, is it
possible.

2024년 1월 9일 (화) 오후 1:58, soomro Shafiollah ***@***.***>님이 작성:

> No problem, thank you!  Ill follow Deepforest repo but it would be very
> helpful if I get thw pertaining code for LiDAR
>
> On Tue, Jan 9, 2024, 1:55 PM Ethan White ***@***.***> wrote:
>
>> Marked as archived now. Sorry for the confusion @safisoomro
>> <https://github.com/safisoomro>
>>
>> —
>> Reply to this email directly, view it on GitHub
>> <https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8065976>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/ALV2GYURFZ6EPXQECLHXOHTYNU43TAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANRVHE3TM>
>> .
>> You are receiving this because you were mentioned.Message ID:
>> ***@***.***>
>>
>

-- 

*Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
*Associate Professor *
*Department of Computer Science*
*Quaid-e-Awam University of Engineering Science and Technology Nawabshah
Sindh Pakistan.*
***@***.*** ***@***.***>*

**Comment by bw4sz** [comment]
Not quite, the LiDAR algorithm is used to generate predictive tree
locations, which are then trained in the RGB crops.

https://www.mdpi.com/2072-4292/11/11/1309

[image: image.png]

So the algorithm takes in RGB and predicts RGB, you could for example
predict in RGB and drape those predictions over a point cloud. If you tell
me more about your use case and need I can direct you to available tools
and data.

On Wed, Jan 10, 2024 at 4:05 AM safisoomro ***@***.***> wrote:

> Hello,
>
> I am wondering if you can show us validation on LiDAR point cloud instead
> of RGB dataset.. because this model is pre-trained on LiDAR data. So, is
> it
> possible.
>
> 2024년 1월 9일 (화) 오후 1:58, soomro Shafiollah ***@***.***>님이 작성:
>
> > No problem, thank you! Ill follow Deepforest repo but it would be very
> > helpful if I get thw pertaining code for LiDAR
> >
> > On Tue, Jan 9, 2024, 1:55 PM Ethan White ***@***.***> wrote:
> >
> >> Marked as archived now. Sorry for the confusion @safisoomro
> >> <https://github.com/safisoomro>
> >>
> >> —
> >> Reply to this email directly, view it on GitHub
> >> <
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8065976>,
>
> >> or unsubscribe
> >> <
> https://github.com/notifications/unsubscribe-auth/ALV2GYURFZ6EPXQECLHXOHTYNU43TAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANRVHE3TM>
>
> >> .
> >> You are receiving this because you were mentioned.Message ID:
> >> ***@***.***>
> >>
> >
>
> --
>
> *Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
> *Associate Professor *
> *Department of Computer Science*
> *Quaid-e-Awam University of Engineering Science and Technology Nawabshah
> Sindh Pakistan.*
> ***@***.*** ***@***.***>*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8079314>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJHBLHCFKRX5VNWPG5ENNTYNZ7ZPAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANZZGMYTI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Ben Weinstein, Ph.D.
Research Scientist
University of Florida

**Comment by safisoomro** [comment]
Yes, thank you the details are here.

My organization has hired me to segment trees from LiDAR data initially.
They are going to provide me data labeled or unlabeled don't know yet.
Meanwhile, I am looking for methods that can perform this task and make
things easier for me, I found your framework interesting but I'll have to
deal with liDAR data, not RGB.

Can you now suggest me where can i begin from...

2024년 1월 11일 (목) 오전 4:10, Benjamin Weinstein ***@***.***>님이
작성:

> Not quite, the LiDAR algorithm is used to generate predictive tree
> locations, which are then trained in the RGB crops.
>
> https://www.mdpi.com/2072-4292/11/11/1309
>
> [image: image.png]
>
> So the algorithm takes in RGB and predicts RGB, you could for example
> predict in RGB and drape those predictions over a point cloud. If you tell
> me more about your use case and need I can direct you to available tools
> and data.
>
> On Wed, Jan 10, 2024 at 4:05 AM safisoomro ***@***.***> wrote:
>
> > Hello,
> >
> > I am wondering if you can show us validation on LiDAR point cloud
> instead
> > of RGB dataset.. because this model is pre-trained on LiDAR data. So, is
> > it
> > possible.
> >
> > 2024년 1월 9일 (화) 오후 1:58, soomro Shafiollah ***@***.***>님이 작성:
> >
> > > No problem, thank you! Ill follow Deepforest repo but it would be very
> > > helpful if I get thw pertaining code for LiDAR
> > >
> > > On Tue, Jan 9, 2024, 1:55 PM Ethan White ***@***.***> wrote:
> > >
> > >> Marked as archived now. Sorry for the confusion @safisoomro
> > >> <https://github.com/safisoomro>
> > >>
> > >> —
> > >> Reply to this email directly, view it on GitHub
> > >> <
> >
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8065976>,
>
> >
> > >> or unsubscribe
> > >> <
> >
> https://github.com/notifications/unsubscribe-auth/ALV2GYURFZ6EPXQECLHXOHTYNU43TAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANRVHE3TM>
>
> >
> > >> .
> > >> You are receiving this because you were mentioned.Message ID:
> > >> ***@***.***>
> > >>
> > >
> >
> > --
> >
> > *Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
> > *Associate Professor *
> > *Department of Computer Science*
> > *Quaid-e-Awam University of Engineering Science and Technology Nawabshah
> > Sindh Pakistan.*
> > ***@***.*** ***@***.***>*
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8079314>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAJHBLHCFKRX5VNWPG5ENNTYNZ7ZPAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANZZGMYTI>
>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
> Ben Weinstein, Ph.D.
> Research Scientist
> University of Florida
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8089541>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALV2GYXXIKED5PW3TWNY4G3YN5JZDAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAOBZGU2DC>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
*Associate Professor *
*Department of Computer Science*
*Quaid-e-Awam University of Engineering Science and Technology Nawabshah
Sindh Pakistan.*
***@***.*** ***@***.***>*

**Comment by bw4sz** [comment]
For sure. This is the LiDAR benchmark, start by training models from there.
Look for the papers that have used this dataset. Terrestrial or airborne?

https://arxiv.org/abs/2309.01279


On Fri, Jan 12, 2024 at 9:09 AM safisoomro ***@***.***> wrote:

> Yes, thank you the details are here.
>
> My organization has hired me to segment trees from LiDAR data initially.
> They are going to provide me data labeled or unlabeled don't know yet.
> Meanwhile, I am looking for methods that can perform this task and make
> things easier for me, I found your framework interesting but I'll have to
> deal with liDAR data, not RGB.
>
> Can you now suggest me where can i begin from...
>
> 2024년 1월 11일 (목) 오전 4:10, Benjamin Weinstein ***@***.***>님이
> 작성:
>
> > Not quite, the LiDAR algorithm is used to generate predictive tree
> > locations, which are then trained in the RGB crops.
> >
> > https://www.mdpi.com/2072-4292/11/11/1309
> >
> > [image: image.png]
> >
> > So the algorithm takes in RGB and predicts RGB, you could for example
> > predict in RGB and drape those predictions over a point cloud. If you
> tell
> > me more about your use case and need I can direct you to available tools
> > and data.
> >
> > On Wed, Jan 10, 2024 at 4:05 AM safisoomro ***@***.***> wrote:
> >
> > > Hello,
> > >
> > > I am wondering if you can show us validation on LiDAR point cloud
> > instead
> > > of RGB dataset.. because this model is pre-trained on LiDAR data. So,
> is
> > > it
> > > possible.
> > >
> > > 2024년 1월 9일 (화) 오후 1:58, soomro Shafiollah ***@***.***>님이 작성:
> > >
> > > > No problem, thank you! Ill follow Deepforest repo but it would be
> very
> > > > helpful if I get thw pertaining code for LiDAR
> > > >
> > > > On Tue, Jan 9, 2024, 1:55 PM Ethan White ***@***.***> wrote:
> > > >
> > > >> Marked as archived now. Sorry for the confusion @safisoomro
> > > >> <https://github.com/safisoomro>
> > > >>
> > > >> —
> > > >> Reply to this email directly, view it on GitHub
> > > >> <
> > >
> >
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8065976>,
>
> >
> > >
> > > >> or unsubscribe
> > > >> <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ALV2GYURFZ6EPXQECLHXOHTYNU43TAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANRVHE3TM>
>
> >
> > >
> > > >> .
> > > >> You are receiving this because you were mentioned.Message ID:
> > > >> ***@***.***>
> > > >>
> > > >
> > >
> > > --
> > >
> > > *Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
> > > *Associate Professor *
> > > *Department of Computer Science*
> > > *Quaid-e-Awam University of Engineering Science and Technology
> Nawabshah
> > > Sindh Pakistan.*
> > > ***@***.*** ***@***.***>*
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8079314>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/AAJHBLHCFKRX5VNWPG5ENNTYNZ7ZPAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANZZGMYTI>
>
> >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> > Ben Weinstein, Ph.D.
> > Research Scientist
> > University of Florida
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8089541>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/ALV2GYXXIKED5PW3TWNY4G3YN5JZDAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAOBZGU2DC>
>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
>
> *Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
> *Associate Professor *
> *Department of Computer Science*
> *Quaid-e-Awam University of Engineering Science and Technology Nawabshah
> Sindh Pakistan.*
> ***@***.*** ***@***.***>*
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8112423>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJHBLDZKW7KDVEA4J5WXHTYOFU27AVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCMJSGQZDG>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Ben Weinstein, Ph.D.
Research Scientist
University of Florida

**Comment by safisoomro** [comment]
I need some help from you.

I have raster images along with CSV data. I am supposed to predict tree
height and stem diameter.
Now can you tell me how I can process these Lidars scans?

>  Am I supposed to view these files in ArcGIS then convert them in CHM and
> label them manually? please let me know.


2024년 1월 12일 (금) 오후 8:19, Benjamin Weinstein ***@***.***>님이
작성:

> For sure. This is the LiDAR benchmark, start by training models from
> there.
> Look for the papers that have used this dataset. Terrestrial or airborne?
>
> https://arxiv.org/abs/2309.01279
>
>
> On Fri, Jan 12, 2024 at 9:09 AM safisoomro ***@***.***> wrote:
>
> > Yes, thank you the details are here.
> >
> > My organization has hired me to segment trees from LiDAR data initially.
> > They are going to provide me data labeled or unlabeled don't know yet.
> > Meanwhile, I am looking for methods that can perform this task and make
> > things easier for me, I found your framework interesting but I'll have
> to
> > deal with liDAR data, not RGB.
> >
> > Can you now suggest me where can i begin from...
> >
> > 2024년 1월 11일 (목) 오전 4:10, Benjamin Weinstein ***@***.***>님이
> > 작성:
> >
> > > Not quite, the LiDAR algorithm is used to generate predictive tree
> > > locations, which are then trained in the RGB crops.
> > >
> > > https://www.mdpi.com/2072-4292/11/11/1309
> > >
> > > [image: image.png]
> > >
> > > So the algorithm takes in RGB and predicts RGB, you could for example
> > > predict in RGB and drape those predictions over a point cloud. If you
> > tell
> > > me more about your use case and need I can direct you to available
> tools
> > > and data.
> > >
> > > On Wed, Jan 10, 2024 at 4:05 AM safisoomro ***@***.***> wrote:
> > >
> > > > Hello,
> > > >
> > > > I am wondering if you can show us validation on LiDAR point cloud
> > > instead
> > > > of RGB dataset.. because this model is pre-trained on LiDAR data.
> So,
> > is
> > > > it
> > > > possible.
> > > >
> > > > 2024년 1월 9일 (화) 오후 1:58, soomro Shafiollah ***@***.***>님이 작성:
> > > >
> > > > > No problem, thank you! Ill follow Deepforest repo but it would be
> > very
> > > > > helpful if I get thw pertaining code for LiDAR
> > > > >
> > > > > On Tue, Jan 9, 2024, 1:55 PM Ethan White ***@***.***> wrote:
> > > > >
> > > > >> Marked as archived now. Sorry for the confusion @safisoomro
> > > > >> <https://github.com/safisoomro>
> > > > >>
> > > > >> —
> > > > >> Reply to this email directly, view it on GitHub
> > > > >> <
> > > >
> > >
> >
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8065976>,
>
> >
> > >
> > > >
> > > > >> or unsubscribe
> > > > >> <
> > > >
> > >
> >
> https://github.com/notifications/unsubscribe-auth/ALV2GYURFZ6EPXQECLHXOHTYNU43TAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANRVHE3TM>
>
> >
> > >
> > > >
> > > > >> .
> > > > >> You are receiving this because you were mentioned.Message ID:
> > > > >> ***@***.***>
> > > > >>
> > > > >
> > > >
> > > > --
> > > >
> > > > *Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
> > > > *Associate Professor *
> > > > *Department of Computer Science*
> > > > *Quaid-e-Awam University of Engineering Science and Technology
> > Nawabshah
> > > > Sindh Pakistan.*
> > > > ***@***.*** ***@***.***>*
> > > >
> > > > —
> > > > Reply to this email directly, view it on GitHub
> > > > <
> > >
> >
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8079314>,
>
> >
> > >
> > > > or unsubscribe
> > > > <
> > >
> >
> https://github.com/notifications/unsubscribe-auth/AAJHBLHCFKRX5VNWPG5ENNTYNZ7ZPAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DANZZGMYTI>
>
> >
> > >
> > > > .
> > > > You are receiving this because you were mentioned.Message ID:
> > > > ***@***.***>
> > > >
> > >
> > >
> > > --
> > > Ben Weinstein, Ph.D.
> > > Research Scientist
> > > University of Florida
> > >
> > > —
> > > Reply to this email directly, view it on GitHub
> > > <
> >
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8089541>,
>
> >
> > > or unsubscribe
> > > <
> >
> https://github.com/notifications/unsubscribe-auth/ALV2GYXXIKED5PW3TWNY4G3YN5JZDAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DAOBZGU2DC>
>
> >
> > > .
> > > You are receiving this because you were mentioned.Message ID:
> > > ***@***.***>
> > >
> >
> >
> > --
> >
> > *Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
> > *Associate Professor *
> > *Department of Computer Science*
> > *Quaid-e-Awam University of Engineering Science and Technology Nawabshah
> > Sindh Pakistan.*
> > ***@***.*** ***@***.***>*
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <
> https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8112423>,
>
> > or unsubscribe
> > <
> https://github.com/notifications/unsubscribe-auth/AAJHBLDZKW7KDVEA4J5WXHTYOFU27AVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCMJSGQZDG>
>
> > .
> > You are receiving this because you were mentioned.Message ID:
> > ***@***.***>
> >
>
>
> --
> Ben Weinstein, Ph.D.
> Research Scientist
> University of Florida
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/weecology/DeepForest/discussions/593#discussioncomment-8113559>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ALV2GYXJ5TIEUBUPKHTVGKLYOGEFFAVCNFSM6AAAAABBKIXDMOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCMJTGU2TS>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*Dr. Shafiullah SoomroPhD, Application Software (South Korea)*
*Associate Professor *
*Department of Computer Science*
*Quaid-e-Awam University of Engineering Science and Technology Nawabshah
Sindh Pakistan.*
***@***.*** ***@***.***>*

**Reply by ethanwhite** [reply]
We really don't work actively with raw LIDAR data (when we do work with LIDAR it is provided as already processed Canopy Height Models) so we can't really help you with that aspect of things. If you end up working primarily with RGB data at some point and want help working with DeepForest let us know.
